src/behavior.ts

```diff
@@ -97,7 +94,9 @@ export abstract class Behavior<A> extends Reactive<A>
   }
   push(a: any): void {
     this.last = this.pull();
-    this.child.push(this.last);
+    for (const child of this.children) {
+      child.push(this.last);
+    }
```

Comment:
These three lines occur a lot. Adding this as a method would avoid the repetition.

Reply:
Yes, I've considered it but did not do it, in hope that it would give better performance.